cmd/flowrunner/testdata/default.json

```diff
@@ -53,7 +53,7 @@
                 "name": "Kigali City Members"
             },
             {
-                "uuid": "d7ff4872-9238-452f-9d38-2f558fea89e0",
+                "uuid": "bf282a79-aa74-4557-9932-22a9b3bce537",
```

Comment:
the additional group change logging showed we were getting this mixed up because it had the same UUID as another group